query-engine/query-engine-wasm/Cargo.toml

```diff
@@ -46,3 +46,15 @@ tracing-subscriber = { version = "0.3" }
 tracing-futures = "0.2"
 tracing-opentelemetry = "0.17.3"
 opentelemetry = { version = "0.17"}
+
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = [
+    "-Oz",                                # super focus on code size
```

Comment:
How much do we win on `-Oz` comapring to `-Os`? Last time we looked into it, size win was quite marginal but performance hit was quite big. I'd rather us do this flag separately, when we are ready to run the benchmarks.

Reply:
WIth `-0s`, we'd have:

```
946B   README.md           
355B   package.json        
28.5kB query_engine_bg.js  
3.4MB  query_engine_bg.wasm
4.7kB  query_engine.d.ts   
538B   query_engine.js     
=== Tarball Details === 
name:          @prisma/query-engine-wasm 
package size:  1.3 MB                                  
unpacked size: 3.4 MB
```

So, still 1.3MB. At some point we'll need to re-run the benchmarks to ensure we aren't optimising too much for size while worsening the performance.

Reply:
I'd prefer we keep it at `-Os` now then and revisit later, in isolation